tools/eslint-config-web3-base/ts-jest.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+	parser: '@typescript-eslint/parser',
+	plugins: ['@typescript-eslint', 'jest'],
+	extends: ['./ts', 'plugin:jest/recommended', 'plugin:jest/style'],
+	env: {
+		'jest/globals': true,
+	},
+	rules: {
```

Comment:
I'm not sure about all these rules being turned off. I feel like we could turn most of them back on and toggle off the ones we need to while developing

Reply:
I updated the rules for the jest, but kept a bit flexibility which I learned with experience is required to to write Jest tests more quickly.